scripts/pytest-utils.sh

```diff
@@ -49,3 +49,33 @@ run_tutorial_test() {
   echo
   python3 -u "$1.py" || $TRITON_TEST_IGNORE_ERRORS
 }
+
+capture_runtime_env() {
+    mkdir -p "$TRITON_TEST_REPORTS_DIR"
+
+    echo "$CMPLR_ROOT" > $TRITON_TEST_REPORTS_DIR/cmplr_version.txt
+    echo "$MKLROOT" > $TRITON_TEST_REPORTS_DIR/mkl_version.txt
+
+    # Exit script execution as long as one of those components is not found.
+    local TRITON_COMMIT=""
+    if [ -d "${SCRIPTS_DIR}/../python/dist" ]; then
+        WHEEL=$(ls ${SCRIPTS_DIR}/../python/dist/*.whl)
+        if [[ -n "$WHEEL" ]]; then
+            TRITON_COMMIT=$(echo "$WHEEL" | sed -n 's/.*git\([a-zA-Z0-9]*\)[^a-zA-Z0-9].*/\1/p')
+        fi
+    fi
```

Comment:
This is simpler and also handles a case when there is more than one *.whl file (possible in a development environment):
```suggestion
    WHEEL=$SCRIPTS_DIR/../python/dist/*.whl
    if [[ -f "$WHEEL" ]]; then
        TRITON_COMMIT=$(echo "$WHEEL" | sed -n 's/.*git\([a-zA-Z0-9]*\)[^a-zA-Z0-9].*/\1/p')
    fi
```

Reply:
I wonder how this change solves a case when there exists more then one *.whl? Like `Triron+git1s2d3f.whl` and `Triron+git4g5h6j.whl` ? 
Which commit is the one?
How about this change:
```suggestion
    WHEELS=($SCRIPTS_DIR/../python/dist/*.whl)
    if [[ "${#WHEELS[@]}" -gt 1 ]]; then
        echo "WARNING: Found multiple Triton whls, please leave the one on which Triton was build"
        exit 1
    fi
    if [[ "${#WHEELS[@]}" -eq 1 ]]; then
        TRITON_COMMIT=$(echo "${WHEELS[0]}" | sed -n 's/.*git\([a-zA-Z0-9]*\)[^a-zA-Z0-9].*/\1/p')
    fi
```